css/mediaqueries/custom-001.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+	<head>
+		<title>Test: support for custom media queries</title>
```

Comment:
It might be nice to pair this test with a set of 5 similar things that all *don't* match.  (For example, a custom query that isn't defined, a `false` one, the `not` of a `true` one, etc.

Reply:
I've added this in a separate file (and shifted all the file numbers).